data-lineage/src/main/java/com/wipro/ats/bdre/lineage/entiity/Table.java

```diff
@@ -125,11 +132,12 @@ public String toDotString() {
 //				middle += "<f" + (i+1) + ">" + column.getColumnName();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)